manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -43,7 +43,6 @@ spec:
             - "--csi-address=$(ADDRESS)"
             - "--timeout=3m"
             - "--default-fstype=ext4"
-            - "--feature-gates=Topology=true"
```

Comment:
@jichenjc Is this feature gate totally removed now?

Reply:
it's by default true now, correct? thus no need to set it ?

Reply:
It says disabled by default? https://github.com/kubernetes-csi/external-provisioner/blob/master/README.md#other-recognized-arguments

Reply:
@jichenjc could you please also add the reference link in the PR description?

Reply:
@jichenjc Topology is off by default. See https://github.com/kubernetes-csi/external-provisioner/blob/master/pkg/features/features.go#L46

But I would be +1 for the log changes.